### PR TITLE
Remove pace js

### PR DESCRIPTION
**Pull request by itkg-alavieille**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1336
https://github.com/open-orchestra/open-orchestra/pull/780

### CODE REVIEW

File: BackofficeBundle/Resources/public/coffee/ajaxLoader/AjaxLoaderView.coffee [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ * @class AjaxLoaderView
+###
+class OpenOrchestra.AjaxLoader.AjaxLoaderView extends Backbone.View
+
+  tagName : 'div'
+
+  className: 'oo-ajax-loader'
+
+  ###*
+   * required options
```
Comment (itkg-ngilain) [inline BackofficeBundle/Resources/public/coffee/ajaxLoader/AjaxLoaderView.coffee]: If an option is required, it's no more an option but a parameter, isn't it ? ^^

**Comment by itkg-ngilain** [comment]
:+1: with or without the "required option" mod

File: BackofficeBundle/Resources/public/coffee/ajaxLoader/AjaxLoaderView.coffee [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   * {
+   *  listenElement : {object} Jquery Element
+   * }
+   * @param {Object} options
+  ###
+  initialize: (options) ->
+    viewContext = @
+    @hideLoader(@);
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/public/coffee/ajaxLoader/AjaxLoaderView.coffee]: you could also throw an exception if the required parameters is not given

**Comment by itkg-alavieille** [comment]
@nicolasThal @itkg-ngilain

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**